packages/core/tests/credentials-auto-accept.test.ts

```diff
@@ -278,7 +302,15 @@ describe('auto accept credentials', () => {
         createdAt: expect.any(Date),
         offerMessage: expect.any(Object),
         requestMessage: expect.any(Object),
-        metadata: { requestMetadata: expect.any(Object) },
+        metadata: {
+          data: {
+            indyRequestMetadata: expect.any(Object),
+            indyCredentialMetadata: {
+              schemaId,
+              credentialDefinitionId: credDefId,
+            },
+          },
+        },
```

Comment:
It's a bit confusing that we have `metadata.data.indyRequestMetadata`? Wouldn't be more straightforward to have `metadata.indyRequest`?

Reply:
The data object is there just for structure. My design was that the user could not easily call `metadata.indyRequestMetdata` and do it through the `get`,`set`, etc. Now, the `data` does not remove this, but I do think it promotes the usage of the functions to get values.

renaming `indyRequestMetdata` to `indyRequest` is okay with me. It does seem a bit too verbose now that I think about it.

Reply:
I see. I think it would be more concise to use just `indyRequest` instead of `indyRequestMetdata` at least.